src/framework/opset.hpp

```diff
@@ -216,27 +221,33 @@ OpSet OpSet::difference(const OpSet &_opset) const {
 // Return a set of all optypes in set not contained in the OpSet
 OpSet::optypeset_t OpSet::difference(const optypeset_t &_optypes) const {
   optypeset_t ret;
-  std::set_difference(_optypes.begin(), _optypes.end(),
-                      optypes.begin(), optypes.end(),
-                      std::inserter(ret, ret.begin()));
+  for (const auto& item : _optypes) {
+    if (optypes.count(item) == 0) {
+      ret.insert(item);
+    }
+  }
```

Comment:
as we use it several times, If it is easy maybe we can put this in a function?
```
template <typename T>
std::unordered_set<T> set_difference_unordered(const std::unordered_set<T>& a, const std::unordered_set<T>& b){ ... }
```
(you may need the Hash template parameter as well to make it work with `optypeset_t`s)

and have only:
```
OpSet::optypeset_t OpSet::difference(const optypeset_t &_optypes) const {
   return set_difference_unordered(_optypes, optypes);
}

```